src/plugins/intel_gpu/src/plugin/transformations_pipeline.cpp

```diff
@@ -158,17 +160,17 @@ static bool is_non_decompression_multiply(const std::shared_ptr<const ov::Node>
     if (!consumer)
         return true;
 
-    if (ov::is_type<ov::opset1::MatMul>(consumer)) {
+    if (ov::is_type<ov::opset1::MatMul>(consumer) || ov::is_type<ov::opset10::Gather>(consumer)) {
```

Comment:
please use operation version instead of opset for consistency.

Reply:
I replaced `opset10` with `op::v8`.